src/lib/Form/DataTransformer/DateIntervalTransformer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\DataTransformer;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+/**
+ * Translates Content's ID to domain specific VersionInfo object.
+ */
+class DateIntervalTransformer implements DataTransformerInterface
+{
+    /**
+     * @param VersionInfo|null $value
+     *
+     * @return array|null
+     *
+     * @throws TransformationFailedException
+     */
+    public function transform($value)
+    {
+        return null;
+    }
+
+    /**
+     * @param array|null $value
+     *
+     * @return array|null
+     *
+     * @throws \Exception
+     * @throws TransformationFailedException
+     * @throws UnauthorizedException
+     * @throws NotFoundException
+     */
+    public function reverseTransform($value)
```

Comment:
Missing type definition? 
Shouldn't it be like: `public function reverseTransform(?Array $value)`

Reply:
We have to change it. I'm using icons in the React modules.

Reply:
I believe you wanted to answer to the comment about modal.
So this is outside the scope of this PR. We should create a ticket and restore the consistency of close button in modals.

Reply:
Sorry and yes